app/src/main/java/org/mozilla/fenix/settings/quicksettings/WebsitePermissionsView.kt

```diff
@@ -55,7 +55,12 @@ class WebsitePermissionsView(
      * @param state [WebsitePermissionsState] to be rendered.
      */
     fun update(state: WebsitePermissionsState) {
-        if (state.isVisible) {
+        val isAnyPermissionVisible = state.camera.isVisible || state.location.isVisible ||
+                state.microphone.isVisible || state.notification.isVisible
```

Comment:
I think we are doing the same here https://github.com/mozilla-mobile/fenix/blob/81d0db76e9d06678f528cdd1db23d7f70f1c0a3f/app/src/main/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsFragmentStore.kt#L147

It will be better if we have the logic centralized in just one place, in [createWebsitePermissionState](https://github.com/mozilla-mobile/fenix/blob/81d0db76e9d06678f528cdd1db23d7f70f1c0a3f/app/src/main/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsFragmentStore.kt#L147) we can just ignore `autoplayAudiblePermission` and `autoplayInaudiblePermission`  as we are not showing them in the panel, this way we can keep the code as it is and have the logic only there. What do you think ? :)

Reply:
Sounds good! I wasn't sure if removing the audio permissions there would have an unintended effect anywhere else. I will push the change.

Reply:
Great job 👍